worlds/hexcellsinfinite/items.py

```diff
@@ -0,0 +1,39 @@
+from BaseClasses import Item, ItemClassification
+from .types import ItemData, HexcellsInfiniteItem
+from .options import LevelUnlockType
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from . import HexcellsInfiniteWorld
+
+def create_itempool(world: "HexcellsInfiniteWorld") -> list[Item]:
+    itempool: list[Item] = []
+
+    if world.options.LevelUnlockType == LevelUnlockType.option_vanilla:
+        itempool.extend(create_multiple_items(world, "Gem", 36))
+    elif world.options.LevelUnlockType == LevelUnlockType.option_individual:
+        for item in item_table.keys():
+            if item != "Gem":
+                itempool.append(create_item(world, item))
+
+    return itempool
+
+def create_item(world: "HexcellsInfiniteWorld", name: str) -> Item:
+    data = item_table[name]
+    return HexcellsInfiniteItem(name, data.classification, data.ap_code, world.player)
+
+def create_multiple_items(world: "HexcellsInfiniteWorld", name: str, count: int,
```

Comment:
Probably a skeleton thing, but why does this function exist separately from `create_item` and also not actually use `create_item` in it? And why does it not use the associated `ItemData` like that one does?